kafka-streams/src/main/java/StreamsFilterBalance2.java

```diff
@@ -130,6 +140,14 @@ public String apply(String jsonValue) {
 
     }
 
+    protected static List<Double> getLast100Trxns(Map<String, Account> accounts, int x) {
+        return accounts.entrySet().stream()
```

Comment:
it is not clear what does this method do? if it returns list of balances for last 100 accounts? are you sure that entrySet() returning accounts in proper way?

Reply:
yes, it is returning balance amount from last n transactions put to the accounts HashMap